modules/Hoot/ui/sidebar/advancedOpts.js

```diff
@@ -351,7 +385,7 @@ export default class AdvancedOpts {
                         d.send = value !== d.default;
                         if ([ 'double', 'int', 'long' ].indexOf ( d.type ) !== -1 ) {
                             d3.select( `#${d.id}-label-wrao` )
-                                .call(self.notNumber, value);
+                                .call(instance.notNumber, value);
```

Comment:
all these places I use instance was just a way to stop passing `this/that/self` objects around since we set the equivalent to the `instance` variable when you init the class.